framework/src/wifi_manager/wifi_manager_api.c

```diff
@@ -189,13 +189,12 @@ wifi_manager_result_e wifi_manager_scan_ap(wifi_manager_scan_config_s *config)
 {
 	NET_LOGI(TAG, "-->\n");
 	if (config) {
-		if (config->channel < 0 || config->channel > 14) {
+		if (config->channel > 14) {
```

Comment:
There is `WIFIMGR_SSID_LEN` for `ssid_length`.
How about defining the channel length also?